src/test/groovy/edu/ie3/datamodel/io/source/csv/CsvTimeSeriesSourceIT.groovy

```diff
@@ -44,66 +33,45 @@ class CsvTimeSeriesSourceIT extends Specification implements CsvTestDataMeta {
 	CsvTimeSeriesSource source
 
 	def setupSpec() {
-		defaultCoordinate = GeoUtils.DEFAULT_GEOMETRY_FACTORY.createPoint(new Coordinate(7.4116482, 51.4843281))
-		coordinateSource = Mock(IdCoordinateSource)
-		coordinateSource.getCoordinate(5) >> Optional.of(defaultCoordinate)
+		coordinateSource = WeatherTestData.coordinateSource
```

Comment:
I think the shared coordinate source isn't even used anymore, isn't it?

Reply:
you're right - adapted it accordingly